.gitignore

```diff
@@ -0,0 +1,14 @@
+# Gauge - metadata dir
+.gauge
+
+# Gauge - log files dir
+logs
+
+# Gauge - reports dir
+reports
+
+# Gauge - JavaScript node dependencies
+node_modules
+
+# enivronment variables
+.env
```

Comment:
please create `.env.sample` and add all the env variables in that. One who is consuming the repo will see the `.env.sample` and will create his own `.env` file on local

Reply:
Done